Week2/test/maartjes-work.test.js

```diff
@@ -1,9 +1,69 @@
-const { maartjesTasks, maartjesHourlyRate, computeEarnings } = require(`../homework/maartjes-work`);
-
-describe('maartjes_work', () => {
-  test('earnings rounded to euro cents', () => {
-    const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
-    const result = earnings.toFixed(2);
-    expect(result).toBe('373.33');
-  });
-});
+'use strict';
+
+// const { maartjesTasks, maartjesHourlyRate, computeEarnings } = require(`../homework/maartjes-work`);
```

Comment:
The idea of these tests is that they check your code. You should not comment them out. 
Also you don't need to copy/paste the data and functions from the `MAKEME.md`. This code already exist in `maartjes-work.js` and `map-filter.js`. You should implement your solutio there, then the tests can verify that your code does what it should do.

Reply:
I am not sure what should I do now , should I delete everything or just delete extra parts ?

Reply:
You should change the `maartjes-work.test.js` and `map-filter.test.js` files back to how they were and put your solutions in `maartjes-work.js` and `map-filter.js`.

Reply:
Thanks Wilgert for your corrections , i will do that .

Reply:
Hi Wilgert , I rewrite everything , and the  ' npm run test-week2 ' work good.

Test Suites: 2 passed, 2 total
Tests:       2 passed, 2 total
Snapshots:   0 total
Time:        6.431s

Reply:
That looks really good!